kobo/apps/service_health/test_service_health_minimal.py

```diff
@@ -0,0 +1,13 @@
+from django.test import TestCase
+from django.urls import reverse
+
+
+class ServiceHealthMinimalTestCase(TestCase):
+    url = reverse('service-health-minimal')
+
+    def test_service_health_minimal(self):
+        with self.assertNumQueries(0):
```

Comment:
I would add a docstring here explaining what this api is and why it shouldn't have any queries. You can omit the comment "Check that the response content contains" because the documentation for assertContains is available in the django project. So the aim is to explain why our code is doing and why. But we don't need to explain how django works, even if that might be helpful to someone very new. The risk here is documenting too much would actually make the documentation less readable - eventually.